Marlin/src/gcode/config/M575.cpp

```diff
@@ -33,12 +34,13 @@
  *   B<baudrate> - Baud rate (bits per second)
  */
 void GcodeSuite::M575() {
-  const int32_t baud = parser.ulongval('B');
+  const long baud = parser.ulongval('B');
```

Comment:
int32_t max is 2GB (2 billions), should be more than enough...

Reply:
I only changed it for consistency with the serial class. I dont have an issue changing the other end to be uint32 as well and matching them that direction, just defaulted to changing new instead of old code!